tests/tensor/test_math.py

```diff
@@ -1404,6 +1422,11 @@ def test_bool(self):
         assert np.all(i)
 
 
+def test_MaxAndArgmax_deprecated():
+    with pytest.raises(AttributeError):
```

Comment:
Add the specific message we expect for trying to access the `MaxAndArgmax`
```suggestion
    with pytest.raises(AttributeError, match=...):
```

Reply:
added this